### PR TITLE
feat(SCP-3061): Updates GitHub OAuth Token

### PR DESCRIPTION
**Pull request by EthanTPainter**

### CODE REVIEW

**Comment by EthanTPainter** [comment]
Keep Open: Need to test deploys in dev

**Comment by EthanTPainter** [comment]
Tested deploy to dev & it's passing the build & test

File: infrastructure/env.template [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CFT_FILE="pipeline.yaml"
```
Comment (joshuajharris) [inline infrastructure/env.template]: should this be named .env.template.dev?

Comment (EthanTPainter) [reply infrastructure/env.template]: no

Comment (joshuajharris) [reply infrastructure/env.template]: that's good enough for me

**Review by joshuajharris — APPROVED** [review]

File: infrastructure/env.template (hunk 1 above)
Comment (EthanTPainter) [reply infrastructure/env.template]: But I can change it for reference

**Review by andrewc89 — APPROVED** [review]

**Pull request merged**